python basics/string_manipulation.py

```diff
@@ -0,0 +1,88 @@
+#String manipulation
+string = "hello, world!"
+
+#Capitalizes the first letter of the string.
+print('Capitalize:', string.capitalize())
+
+#Replaces a specific part of the string with another.
+#In this case, it replaces every 'hello' with 'Greetings'
+print('Replace:', string.replace('hello', 'Greetings'))
+
+#Capitalizes the first letter of every word.
+print('Title:', string.title())
+
+#Turns every letter into upper case.
+print('Uppercase:', string.upper())
+
+string = 'HELLO, WORLD!'
+
+#Turns every letter into lower case.
+print('Lowercase:', string.lower())
+
+string = ' hello, world! '
+
+#Removes the extra spaces from the right.
+print('Right strip:', string.rstrip())
+
+#Removes the extra spaces from the left.
+print('Left strip:', string.lstrip())
+
+#Removes the extra spaces both from left and right.
+print('Strip:', string.strip())
+
+#Splits the current string into a list of strings.
+#In this case, every list element is separated from another
+#when there's a comma in the original string.
+#If no argument is given, it splits where it finds a blank space.
+string_split = string.split(',')
+print(string_split)
+
+#After splitting, we can merge them together once more using join.
+print('Join:', ''.join(string_split))
+
+#Originally, join inserts a set of characters for every character in a string.
+#You can use it to put - after every character, for example.
+print('Separate characters:', '-'.join(string))
+
+#Or to separate strings from a list of strings when joining them.
+print('Separate strings:', '-'.join(string_split))
+
+#Strings are "a list of characters", and you can access
+#specific parts of a string just like a list.
+string = 'Hello, world!'
```

Comment:
is there a reason you explained what a string is only there? would be better to put it at the beginning of the code, no? :o